### ISSUE TITLE
No refresh after accesing a link

### ISSUE THREAD

**Issue by samarulmeu** [SidOfc/mkdx#134]
Hello! 

Here I am with another issue :)

**OS type**:

- [x] Unix
- [ ] Windows
- [ ] Other ([SPECIFY])

**Vim**:

- [x] vim
- [ ] neovim
- [ ] Other ([SPECIFY])

**Vim version**:

8.1

After I access a link using `gx` the vim *screen* remains dark. And only after I move the cursor around some of the lines are visible again. Or I have to do a `Ctrl-L` for a refresh. It is a mkdx issue as, if I disable the plugin, the `gx` works as expected, no need for refresh. 

Thank you!

**Comment by SidOfc** [comment]
Hey @samarulmeu, cheers for reporting!

I applied a fix for it as well, can you update mkdx and check it out?

**Comment by samarulmeu** [comment]
Wow! That was fast. It is perfect. Thank you!

And closing...
